src/cascadia/TerminalConnection/AzureConnectionStrings.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+// Messages to the user
+const auto resource = L"Terminal";

```

Comment:
These all need to go into a resource file. Good job centralizing them, but we need to show you how to use the resource loader.